build/charts/antrea/templates/agent/clusterrole.yaml

```diff
@@ -219,3 +218,15 @@ rules:
     - get
     - list
     - watch
+  - apiGroups:
+      - crd.antrea.io
+    resources:
+      - trafficcontrols
+    verbs:
+      - get
+      - watch
+      - list
+      - update
+      - patch
+      - create
+      - delete
```

Comment:
that seems like a lot of permissions for the Agent. A compromised Agent / Node could have the ability to create arbitrary TrafficControl resource, which in my view is an issue. Typically we try to limit such permissions in the Agent.

Reply:
I think perhaps the implementation shouldn't even use TrafficControl to achieve it. It could call the OpenFlow interface `InstallTrafficControlMarkFlows` directly with some predefined parameters, instead of creating a TrafficControl object, which is more complicated. For example, if I annotate only one Pod of a Deployment which has multiple Pods, how can a TrafficControl select that Pod only when all Pods have the same labels?